src/js/models/maps/Map.js

```diff
@@ -344,6 +397,24 @@ define([
         return [];
       },
 
+      /**
+       * Get all of the layers regardless of presences of layerCategories in a
+       * flat list of MapAsset models.
+       * @returns {MapAsset[]} All of the layers, or empty array if no layers
+       * are configured.
+       */
+      getAllLayers() {
```

Comment:
I'm kind of wondering whether the Map model should set up a MapAssets collection during init that includes all the layers regardless of whether they are in categories or not.